docs/content/integrations/airlift/airflow-1-migration.mdx

```diff
@@ -0,0 +1,264 @@
+# Airflow 1 Migration Tutorial
```

Comment:
nit: is the standard to say `airflow 1` or `airflow 1.x`?

Reply:
I'll say 1.X just to be explicit.